config/initializers/cache_store.rb

```diff
@@ -11,6 +11,7 @@ def self.memcached
           username: ENV["MEMCACHIER_USERNAME"],
           password: ENV["MEMCACHIER_PASSWORD"],
           expires_in: CACHE_EXPIRY,
+          value_max_bytes: 10_485_760,
```

Comment:
Thanks for jumping on this ⭐ A couple of things:

1) Can you link to something that shows that Heroku has a 25MB limit?
2) If the limit is 25 MB, why are we only setting the max to ~10.5MB?
3) Can this be tested? E.g. does Heroku allow you to run the rake task from a branch?

Reply:
1. You can see the total memory [here](https://analytics.memcachier.com/#/caches/811908/)
2. As I understand it, this is the total value per item and we're storing [3 items](https://github.com/search?q=repo%3Aalphagov%2Fgovuk-dependencies%20cache.fetch&type=code) but I only expect the unreviewed PRs one to ever be larger than 1MB, so the 10x increase in `value_max_bytes` was just one that I picked arbitrarily.
3. I've deployed this to Heroku and ran the rake task again, which resulted in Dependapanda posting to slack channels.

Reply:
It seems like this approach won't work: https://devcenter.heroku.com/articles/memcachier#key-value-size-limit-1mb

I think we have to finally take the time to refactor this app to make sensible API calls and stop caching.